lerobot/configs/policy/hilserl_classifier.yaml

```diff
@@ -39,7 +39,6 @@ policy:
 wandb:
   enable: false
   project: "classifier-training"
-  entity: "wandb_entity"
```

Comment:
I have removed this param, because if you launch the training script it will crashes if `wandb` is enabled. That happen because you need setup this parameter correctly. It's a name of your namespace in the `wandb`. I was strugling around it, as the error generated by the service was saying that I don't have permissions for the bucket. By default `wandb` will use user's namespace and it will work fine.